content/doc/book/pipeline/index.adoc

```diff
@@ -6,43 +6,150 @@ layout: chapter
+
+
+While Jenkins has always allowed rudimentary forms of chaining Freestyle Jobs
+together to perform sequential tasks,
+footnote:[Additional plugins have been used to implement complex behaviors
+utilizing Freestyle Jobs such as the Copy Artifact, Parameterized Trigger,
+and Promoted Builds plugins]
+Pipeline makes this concept a first-class citizen in Jenkins.
+
+Building on the core Jenkins value of extensibility, Pipeline is also
+extensible both by users with <<pipeline/shared-libraries#,Pipeline Shared Libraries>>
+and by plugin developers.
+footnoteref:[ghof,link:https://plugins.jenkins.io/github-organization-folder[GitHub
+Organization Folder plugin]]
+
+
+The flowchart below is an example of one continuous delivery scenario easily
+modeled in Jenkins Pipeline:
+
+image::/images/pipeline/realworld-pipeline-flow.png[title="Pipeline Flow", 800]
+
+
+[[terms]]
+== Pipeline Terms
 
-If you are a Jenkins administrator and want to know more about managing Jenkins nodes and instances, see
-<<managing#,Managing Jenkins>>.
+[[step]]
+Step::
+    A single task, fundamentally steps tell Jenkins _what_ to do. For example, in
+    order to instruct Jenkins to execute the shell command `make`, the `sh`
+    step would be used: `sh 'make'`.
+    When a plugin extends to Pipeline DSL, that typically means the plugin has
+    implemented a new _step_.
 
-If you are a system administrator and want learn how to back-up, restore, maintain as Jenkins servers and nodes, see
-<<operating#,Operating Jenkins>>.
+[[node]]
+Node::
+    Generally speaking, all _work_ which is done by a Pipeline, will be done in
+    the context of a single, or multiple, `node` step declarations, which does two things:
+    . Schedules the steps contained within the block to run by adding an item
+      to the Jenkins queue. As soon as an executor is free on a node, the
+      steps will run.
+    . Creates a workspace, a directory specific to that particular
+      Pipeline, where files can be checked out from source control and work can
+      be done.
 
+CAUTION: Depending on your Jenkins configuration, some workspaces may not get
+automatically cleaned up after a period of inactivity. See tickets and
+discussion linked from
+https://issues.jenkins-ci.org/browse/JENKINS-2111[JENKINS-2111]
+for more information.
 
-[WARNING]
-====
-*To Contributors*:
-This chapter functions as a continuation of "<<getting-started#,Getting Started with Jenkins>>"
-and "<<using#,Using Jenkins>>", but the format will be slightly different - see the description above.
-The first sections should lead users through the basics of pipeline, and later sections can switch to
-feature reference for experienced users. All sections should still
-be written and ordered to only assume knowledge from "Getting Started", "Using Jenkins", or
-from previous sections in this chapter.
-====
+[[stage]]
+Stage::
+    A "stage" is a step for defining a conceptually distinct subset of the
```

Comment:
Should this be a "_stage_" to reference the DSL stage step or did you want to define what a stage (noun) is?